sdk/iot/doc/mqtt_state_machine.md

```diff
@@ -146,6 +155,8 @@ _Example:_
     }
 ```
 
+__Important:__ C2D messages are not enqueued until the device establishes the first MQTT session (connects for the first time to IoT Hub). The C2D message queue is preserved (according to the per-message time-to-live) as long as the device connects with Clean Session `false`.

```

Comment:
What is `Clean Session`? Is it a term that folks familiar with the IoT Hub domain would be aware off, or is it part of our API surface? Consider adding a link to a reference.

Reply:
I won't add it here though: it has its own paragraph under Connecting.